### ISSUE TITLE
Table: VirtualScrolling blocked by table in other TabPanel

### ISSUE THREAD

**Issue by otto-consid** [primefaces/primeng#13257]
### Describe the bug

When using tables in tabs, the available lines to scroll a table in a prior tab is limited by an unrendered tables in subsequent tabs.
This occurs when using virtual scrolling and when the content has not loaded yet. If all subsequent (limiting) tabs are opened (rendered) the scrolling behaves as expect again.
The same bug also appears when Accordion is used to hide tables.

### Environment

Windows

### Reproducer

https://stackblitz.com/edit/nbjxem

### Angular version

16.0.0

### PrimeNG version

16.0.2

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.20.0

### Browser(s)

Firefox 114

### Steps to reproduce the behavior

1. Go to [https://stackblitz.com/edit/nbjxem?file=src%2Fapp%2Fdemo%2Ftable-virtual-scroll-demo.html](https://stackblitz.com/edit/nbjxem?file=src%2Fapp%2Fdemo%2Ftable-virtual-scroll-demo.html)
2. In the first tab (Long Table), scroll the table down. 
3. The table keeps jumping back up causing flickering and blocking access to subsequent rows.
4. Visit second tab (Long Table Again) and Go back to first tab (Long Table) again.
5. Scrolling now works as expected.

### Expected behavior

A table with virtual scrolling to scroll smoothly regardless of unrendered or uncalculated content in other tables, tabs or .

**Comment by Phil-DS** [comment]
I also have experienced a similar issue. I had 2 virtual scroll tables, and it was triggering double scroll events when I scrolled on one table. My workaround was to *ngIf the tables, to prevent them showing. This is a painful issue

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
